### DISCUSSION TITLE
Split apk?

### DISCUSSION THREAD

**Discussion by ghost** [android-hacker/VAExposed#985] [Q&A] [answered]
@tiann  how do i instal:
Apks
Apkm
Xapk 

Inside VXP 0.20.3?

You say it support split apk

**Comment by tiann** [comment, marked as answer]
Only Google's split apks is supported, third-party(Xapk, apkm) won't work

**Reply by ghost** [reply]
What?where i can download it?
@tiann

**Reply by tiann** [reply]
Google play
